### PR TITLE
(feat) O3-2782: Show an inline notification for camera access errors in the add attachment modal

### PR DESCRIPTION
**Pull request by usamaidrsk**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR adds an inline notification when user has camera disabled when adding attachments by camera, or any other camera errors

## Screenshots

image

## Related Issue

[O3-2782](https://openmrs.atlassian.net/browse/O3-2782)

### CODE REVIEW

**Comment by usamaidrsk** [comment]
@denniskigen please take a look at this provision
Thanks

**Comment by denniskigen** [comment]
Hey, @usamaidrsk. Thanks for working on this. 

Please use the `feat` [conventional commit label](https://www.conventionalcommits.org/en/v1.0.0/#summary) for new features. We don't use an `improvement` label.

File: packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss [hunk 1]
```diff
@@ -38,13 +38,13 @@
...
-    text-align: center;
+    text-align: right;
     color: $ui-01;
     background-color: $danger;
     padding: 0 4px;
     border-radius: 50%;
-    top: -10px;
-    right: -10px;
+    top: -14px;
+    right: -12px;
```
Comment (denniskigen) [inline packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss:47]: Why is this included in the diff?

**Comment by denniskigen** [comment]
Looks good. I think it's preferable to show a custom error message instead of the default error message from the library. More specifically, I think `Please enable camera access in your browser settings and try again` is preferable to:

- CleanShot 2024-02-02 at 2  12 34@2x

File: packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss (hunk 1 above)
Comment (usamaidrsk) [reply packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss:47]: I'm not sure why, because I believe this change was merged in sometime back

Comment (denniskigen) [reply packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss]: I've squashed the commits and removed this from the diff.
